lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -149,8 +149,8 @@ export default class BitcoinProcessor {
   /**
    * Initializes the Bitcoin processor
    */
-  public async initialize () {
-    await this.versionManager.initialize(this.blockMetadataStore);
+  public async initialize (versionModels: BitcoinVersionModel[]) {
```

Comment:
I guess `BitcoinProcessor` was the example I was looking for for dependency injection, so I still want to make sure we have an agreed upon pattern. Let's discuss but won't block the PR on this.